web/custom-plugins/forecast-of-response/src/components/Result.tsx

```diff
@@ -184,25 +187,22 @@ export default (props: { result: any }) => {
         {graphNames.length >= 1 && (
           <AddedGraphWrapper>
             {graphNames.map((graph, index) => (
-              <Chip
-                key={index}
-                style={{ margin: '10px 5px' }}
-                variant="active"
-                onDelete={() => removeGraph(graph.name)}
-              >
-                <IconWrapper color={plotColors[index]}>&#9679;</IconWrapper>
-                {graph.name}
-              </Chip>
+              <Tooltip title={graph.description}>
+                <Chip
+                  key={index}
+                  style={{ margin: '10px 5px', cursor: 'help' }}
+                  variant="active"
+                  onDelete={() => removeGraph(graph.name)}
+                >
+                  <IconWrapper color={plotColors[index]}>&#9679;</IconWrapper>
+                  {graph.name}
+                </Chip>
+              </Tooltip>
             ))}
           </AddedGraphWrapper>
         )}
       </div>
-      <LinesOverTime
-        data={chartData}
-        warningLine={1.2}
-        MaxLine={2.7}
-        graphNames={graphNames}
-      />
+      <LinesOverTime data={chartData} graphNames={graphNames} />
```

Comment:
maybe the state graphNames could be renamed to graphInfo or something (since it now contains more info like description and plotType)

Reply:
agreed. Fixed